lighthouse-core/scripts/lantern/collect/collect.js

```diff
@@ -226,59 +231,70 @@ async function main() {
     // Can run in parallel.
     const wptResultsPromises = [];
     for (let i = 0; i < SAMPLES; i++) {
-      const resultPromise = repeatUntilPass(() => runForWpt(url));
+      const resultPromise = repeatUntilPassOrEmpty(() => runForWpt(url));
```

Comment:
I guess this could be an empty object now, but that's ok since L256 filters those out? maybe a comment to call that out

Reply:
what sort of comment text are you envisioning?

I ask because it's in the method invocation name and hover type and as somehow who didn't write this I had no expectations about what type of result it would be returning anyway :)

Reply:
I'm mainly concerned with it being obvious that we aren't saving empty results.